tools/ete/ete_mod.xml

```diff
@@ -0,0 +1,125 @@
+            truevalue="--resolve_polytomies" falsevalue=""
+            help="Converts polytomies into bifurcations (--resolve_polytomies)" />
+        <param name="standardize" type="boolean" label="Standardize tree"
+            truevalue="--standardize" falsevalue=""
+            help="Standardize tree topology by expanding polytomies and single child nodes (--standardize)" />
+    </inputs>
+    <outputs>
+        <data name="output_tree" format="nhx" label="${tool.name} on ${on_string}"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="outgroup" value="A" />
+            <output name="output_tree" file="tree1_ogA.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="prune" value="A C" />
+            <output name="output_tree" file="tree1_pruneAC.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="prune" value="A C" />
+            <param name="prune_preserve_lengths" value="--prune_preserve_lengths" />
+            <output name="output_tree" file="tree1_pruneACpreserve.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="unroot" value="--unroot" />
+            <output name="output_tree" file="tree1_unroot.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree2.nhx" />
+            <param name="sort_branches" value="--sort_branches" />
+            <output name="output_tree" file="tree2_sort.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="ladderize" value="--ladderize" />
+            <output name="output_tree" file="tree1_ladder.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1_unroot.nhx" />
+            <param name="resolve_polytomies" value="--resolve_polytomies" />
+            <output name="output_tree" file="tree1_unroot_resolve.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree3.nhx" />
+            <param name="standardize" value="--standardize" />
+            <output name="output_tree" file="tree3_stand.nhx" />
+        </test>
+    </tests>
+    <help><![CDATA[
+Modify a given input tree using the etetoolkit: http://etetoolkit.org/
+]]></help>
+    <citations>
+        <citation type="bibtex">
+@article{huerta2016ete,
+  title={ETE 3: reconstruction, analysis, and visualization of phylogenomic data},
+  author={Huerta-Cepas, Jaime and Serra, Fran{\c{c}}ois and Bork, Peer},
+  journal={Molecular Biology and Evolution},
+  volume={33},
+  number={6},
+  pages={1635-1638},
+  year={2016}
+}
```

Comment:
You can replace with:
```
<citation type="doi">10.1093/molbev/msw046</citation>
```

Actually it starts to make sense to have a macro file for requirements and citations.